packages/dtslint-runner/src/main.ts

```diff
@@ -48,8 +48,8 @@ export async function runDTSLint({
   const typesPath = joinPaths(definitelyTypedPath, "types");
 
   const { packageNames, dependents } = onlyRunAffectedPackages
-    ? await prepareAffectedPackages(definitelyTypedPath, nProcesses)
-    : await prepareAllPackages(definitelyTypedPath, definitelyTypedAcquisition.kind === "clone", nProcesses);
+    ? await prepareAffectedPackages(definitelyTypedPath)
```

Comment:
The reference to `nProcesses` went away here but the variable didn't actually go away; is there more cleanup missing?

Reply:
Oh, so it still runs things in parallel? I guess I assumed it all was going away but I guess it's just parallel parse.